src/System.Management.Automation/engine/parser/Parser.cs

```diff
@@ -4514,7 +4514,7 @@ private MemberAst ClassMemberRule(string className, out List<Ast> astsOnError)
                 }
```

Comment:
@vexx32, your last commit had 1 failures in `PowerShell-CI-linux`
Clean up open Runspaces when exit powershell process.PowerShell process should not freeze at exit
```powershell
Expected $true, but got $false.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/Api/BasicEngine.Tests.ps1: line 53
53:         Wait-UntilTrue -sb { $process.HasExited } -TimeoutInMilliseconds 5000 -IntervalInMilliseconds 1000 | Should -BeTrue
```

Reply:
Seems like another transient failure.... very odd.

Reply:
Restarted Macos and Linux CIs.